e2e/txpool_test.go

```diff
@@ -294,39 +294,31 @@ func TestTxPool_RecoverableError(t *testing.T) {
 		})
 		require.NoError(t, err, "Unable to send transaction, %v", err)
 
-		txHash := ethgo.Hash(types.StringToHash(response.TxHash))
-
 		// save for later querying
-		hashes[i] = txHash
+		hashes[i] = types.StringToHash(response.TxHash)
 	}
 
 	ctx, cancel := context.WithTimeout(context.Background(), 6*time.Second)
 	defer cancel()
 
 	// wait for the last tx to be included in a block
-	receipt, err := tests.WaitForReceipt(ctx, client.Eth(), hashes[2])
+	receipt, err := tests.WaitForReceipt(ctx, client, hashes[2])
 	require.NoError(t, err)
 	require.NotNil(t, receipt)
 
 	// assert balance moved
-	balance, err := client.Eth().GetBalance(ethgo.Address(receiverAddress), ethgo.Latest)
+	balance, err := client.GetBalance(receiverAddress, jsonrpc.LatestBlockNumberOrHash)
 	require.NoError(t, err, "failed to retrieve receiver account balance")
 	require.Equal(t, framework.EthToWei(3).String(), balance.String())
 
 	// Query 1st and 2nd txs
-	firstTx, err := client.Eth().GetTransactionByHash(hashes[0])
+	firstTx, err := client.GetTransactionByHash(hashes[0])
 	require.NoError(t, err)
 	require.NotNil(t, firstTx)
 
-	secondTx, err := client.Eth().GetTransactionByHash(hashes[1])
+	secondTx, err := client.GetTransactionByHash(hashes[1])
 	require.NoError(t, err)
 	require.NotNil(t, secondTx)
-
-	// first two are in one block
-	require.Equal(t, firstTx.BlockNumber, secondTx.BlockNumber)
-
-	// last tx is included in next block
-	require.NotEqual(t, secondTx.BlockNumber, receipt.BlockNumber)
```

Comment:
What happened with these assertions?

Reply:
I will revert these. Removed it on the first iteration, and forgot to revert them back.

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/165/commits/6a43df12d00f176d42daa16a06305926806ccfcd